tests/RecordTests/UnitTest/record_ut.cpp

```diff
@@ -129,6 +131,68 @@ TEST_F(record_ut, new_cluster_out_of_order)
     ASSERT_EQ(context->pending_clusters.size(), 3u);
 }
 
+TEST_F(record_ut, DISABLED_bgra_color_max_disk_write)
```

Comment:
why disabled? seems like it could just be 33 frames

Reply:
ahh, I see the purpose, please add a comment to the function say why it is disabled